site/content/en/reference/live/status/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Status"
+linkTitle: "status"
+type: docs
+description: >
+   Status shows the status for the resources in the cluster
+---
+<!--mdtogo:Short
+    Status shows the status for the resources in the cluster
+-->
+
+The status command will print the status for all resources in the live state
+that belong to the current inventory. It will use the inventory template to
```

Comment:
you use a proper noun here: Inventory Template

Is that defined anywhere?

Reply:
Updated this with a link to the apply documentation where we explain the Inventory Template.